### PR TITLE
add config flag for syncing to missed stores

### LINKED ISSUE #3: Restrict get function from making put operations
Hi, this package looks perfect for my use case but there is one problem inside `get` function:

https://github.com/antriver/laravel-multi-cache/blob/7f8b0e2bb789af507629660856d12ee14c4dba8c/src/MultiStore.php#L89-L112

Calling `get` will try to synchonize the key to `missedStores`. This behaviour is not needed in my case, especially with the hard-coded expire of 1 day.

My question is: are you willing to accept a PR adding a config flag that turns on/off this behaviour?

### PR DESCRIPTION
**Pull request by gabriel-vasile**
fixes #3

### CODE REVIEW

**Comment by antriver** [comment]
Hi @gabriel-vasile 

Thanks for this PR. It's a useful change.
There is just one comment I left to address the case when the new config setting has not been defined.
Thanks for adding a test too!

File: src/MultiStore.php [hunk 1]
```diff
@@ -51,6 +56,7 @@ public function __construct(Application $app, array $config, CacheManager $cache
         $this->app = $app;
         $this->config = $config;
         $this->cacheManager = $cacheManager;
+        $this->syncMissedStores = $config['sync_missed_stores'];
```
[the remark this answers is not in the document]
Comment (gabriel-vasile) [reply src/MultiStore.php]: Ah, right. I made the change and I added a test for it.

**Comment by antriver** [comment]
Thanks @gabriel-vasile 
Merged :)

**Comment by antriver** [comment]
Version 10.1.0 has been released containing this change

**Pull request merged**